test-dts/plugins.d-test.ts

```diff
@@ -0,0 +1,46 @@
+import { expectError, expectType } from 'tsd'
+import { ComponentPublicInstance } from 'vue'
+import { config, VueWrapper } from '../src'
+
+interface OptionsI {
+  msg: string
+}
+
+function PluginWithOptions(
+  wrapper: VueWrapper<ComponentPublicInstance>,
+  options: OptionsI
+) {
+  return {}
+}
+
+function PluginWithoutOptions(wrapper: VueWrapper<ComponentPublicInstance>) {
+  return {}
+}
+
+function PluginWithOptionalOptions(
+  wrapper: VueWrapper<ComponentPublicInstance>,
+  options: OptionsI = { msg: 'hi' }
+) {
+  return {}
+}
+
+function PluginWithOptionalOptions2(
+  wrapper: VueWrapper<ComponentPublicInstance>,
+  options?: OptionsI
+) {
+  return {}
+}
+
+config.plugins.VueWrapper.install(PluginWithOptions, { msg: 'Hello' })
+config.plugins.VueWrapper.install(PluginWithoutOptions)
+config.plugins.VueWrapper.install(PluginWithOptionalOptions)
+config.plugins.VueWrapper.install(PluginWithOptionalOptions2)
+config.plugins.VueWrapper.install(PluginWithOptionalOptions, { msg: 'hello' })
+config.plugins.VueWrapper.install(PluginWithOptionalOptions2, { msg: 'hello' })
+
+// FIXME: not sure if this one is possible
+// expectError(config.plugins.VueWrapper.install(PluginWithoutOptions, {}))
```

Comment:
I guess this is fine?
The FIXME needs to be removed before merging.

Reply:
I think it's fine too but I thought I might have overlooked a way of doing it in TS. If there is nothing to fix, yeah, better remove it 😆

Reply:
We _could_ probably do something, but I'm not sure it's worth the time, the hassle and the maintenance cost 😉 Thanks for the PR!